openmp-helloworld/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+project(openmp_helloworld)
+
+cmake_minimum_required(VERSION 3.16)
+
+# Search for rocm in common locations
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)
+
+# Find HIP
+find_package(hip REQUIRED)
+# The user may override AMDGPU_TARGETS defined in the HIP config file
```

Comment:
need to move this before find_package and add pls a note about it in the comments

Reply:
Done, added before find_package, and a note in README